nixos/tests/glances.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, ... }:
+
+{
+  name = "glances";
+
+  nodes = {
+    machine_default =
+      { pkgs, ... }:
+      {
+        services.glances = {
+          enable = true;
+        };
+      };
+
+    machine_custom_port =
+      { pkgs, ... }:
+      {
+        services.glances = {
+          enable = true;
+          port = 5678;
+        };
+      };
+  };
+
+  testScript = ''
+    machine_default.start()
+    machine_default.wait_for_unit("glances.service")
+    machine_default.wait_for_open_port(61208)
+
```

Comment:
would probably require another instance but might be nice to have a test for connecting to another instance

Reply:
Do you now of any modules that does a test like this to use for inspiration? (this is my first contribution)

Reply:
no sorry I'm yet to properly look into the testing system myself this is just a thought that came up from looking over glances myself, thought the ability to monitor another instance was neat and that it was unique enough a use case to justify another test, though there are certainly more reasonable tests to make theres no real need to get held up on them. a nice to have.